lib/components/Button/Button.stories.ts

```diff
@@ -13,34 +13,171 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
-export const Base: Story = {
+const backgrounds = {
+  default: 'jod-bg-gray',
+  values: [{ name: 'jod-bg-gray', value: '#f5f5f5' }],
+};
+
+export const Gray: Story = {
```

Comment:
There doesn't seem to be keyboard focus ring for `gray` and `delete` variants? At least I couldn't see when using keyboard.

Reply:
Intentionally omitted because pseudo classes are not designed for `delete` variant.